HISTORY.rst

```diff
@@ -4,12 +4,16 @@ History
 
 0.36.0 (unreleased)
 -------------------
-Contributors to this version: Pascal Bourgault (:user:`aulemahal`).
+Contributors to this version: Pascal Bourgault (:user:`aulemahal`), Juliette Lavoie (:user:`juliettelavoie`).
 
 Bug fixes
 ^^^^^^^^^
 * Invoking ``lazy_indexing`` twice in row (or more) using the same indexes (using dask) is now fixed. (:issue:`1048`, :pull:`1049`).
 
+New features and enhancements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* `xclim.sdba.measures.rmse` and `xclim.sdba.measures.mae` now use numpy instead of sklearn. This is improves performances when using dask.
```

Comment:
```suggestion
* `xclim.sdba.measures.rmse` and ``xclim.sdba.measures.mae`` now uses `numpy` instead of sklearn. This improves performance when using `dask`. (:pull:`1051`).
```

Reply:
(Sorry for the multiple edits, I didn't see at first that there were two functions mentioned)